src/main/scala/com/geirsson/codegen/Codegen.scala

```diff
@@ -23,16 +23,13 @@ case class Error(msg: String) extends Exception(msg)
 @ProgName("db-codegen")
 case class CodegenOptions(
     @HelpMessage("user on database server") user: String = "postgres",
-    @HelpMessage("password for user on database server") password: String =
-      "postgres",
+    @HelpMessage("password for user on database server") password: String = "postgres",
     @HelpMessage("jdbc url") url: String = "jdbc:postgresql:postgres",
     @HelpMessage("schema on database") schema: String = "public",
-    @HelpMessage("only tested with postgresql") jdbcDriver: String =
-      "org.postgresql.Driver",
+    @HelpMessage("only tested with postgresql") jdbcDriver: String = "org.postgresql.Driver",
     @HelpMessage(
       "top level imports of generated file"
-    ) imports: String = """|import java.util.Date
-                           |import io.getquill.WrappedValue""".stripMargin,
+    ) imports: String = """import io.getquill.WrappedValue""",
```

Comment:
Unrelated, but I just saw quill 1.0 removed the need for `WrappedValue[T]` so this will need to be configurable soon :)